project_charter.md

```diff
@@ -189,6 +189,6 @@ Since official updates require coordination across all platforms, the timeline f
 ## Lifetime of spack-stack installations
 Regular, full releases of spack-stack are made every three months around the end of February, May, August, and November.
 
-spack-stack developers are committed to maintaining a rolling stock of the last four releases, i.e. one year's worth of spack-stack releases. Under special circumstances (e.g. operational implementations that require long-term support, long-running official retrospectives), selected releases may be kept longer.
+spack-stack developers are committed to maintaining a rolling stock of the last four releases, i.e. one year's worth of spack-stack releases. Under special circumstances (e.g. operational implementations that require long-term support, long-running official retrospectives), selected releases may be kept longer. Maintenance in this context referts to addressing bugs or recompiling in the event of an intrusive update to the system. Adding new functionality or new packages is in general limited to the current and future releases.
```

Comment:
add sth like exceptions can be made on a case by case basis?

Reply:
:) see my comment